base.pom.xml

```diff
@@ -168,6 +168,12 @@
             <version>6.0.6</version>
             <scope>test</scope>
         </dependency>
+<!--        <dependency>-->
```

Comment:
@nvoxland Should we remove commented code?

Reply:
I left it in so it's a bit easier to do manual testing without having to re-hunt down what the dependency looks like.